Added Pluto.jl to Browser workflow in docs (#44683)
JuliaLang/julia: doc/src/manual/workflow-tips.md

## Browser-based workflow

> Here are a few ways to interact with the Julia REPL in a browser:
> 
>   * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl).
> 
>   * Another way is through [Pluto](https://github.com/fonsp/Pluto.jl) notebooks.

Reviewer: ```suggestion
There are a few ways to interact with Julia in a browser:
- [IJulia.jl](https://github.com/JuliaLang/IJulia.jl)
- [Pluto.jl](https://github.com/fonsp/Pluto.jl)
```

Reviewer: Similar now to the list in `Profile`: https://raw.githubusercontent.com/JuliaLang/julia/master/doc/src/manual/profile.md

Author: @rikhuijzer I've made those changes. Is it fine now?

Author: @rikhuijzer any idea about when this MR will be merged?

## Browser-based workflow

Here are a few ways to interact with the Julia REPL in a browser:

  * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl). 
    See the package home for details.

  * Another way is via [Pluto](https://github.com/fonsp/Pluto.jl) notebooks. 

>     See the [wiki](https://github.com/fonsp/Pluto.jl/wiki) for details.

Reviewer: Fons is actually working on a new website so this comment will become outdated pretty soon. Maybe remove both suggestion for details and just link to the core GitHub page?

Author: @rikhuijzer done, is it fine now?
